content/docs/custom-plugins/plugin-registration/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Adding Your Custom Plugins to Roadie
+publishedDate: '2023-07-21T12:11:00.0Z'
+description: How to add custom plugins to your Roadie instance
+---
+
+## Introduction
+
+The Roadie CLI builds and packages plugins into a format that is immediately applicable to be registered on your Roadie instance. The CLI provides instructional output on a best-guess basis to help you with plugin registration. 
+
+The components registered on the ‘Custom Plugins’ page are immediately available to be used (after reloading the page). To register plugins to the application you can in most cases use the instructions and output provided by Roadie CLI. 
+
+## Registration Steps
+
+1. Identify the URL where your plugin will be hosted.
+   * In development mode this will usually be `http://localhost:7043`
+   * If you are hosting the plugin on Roadie infrastructure, the URL will be similar to `https://static-assets.roadie.so/<your-tenant-name>/<your-plugin-name>`
+2. Identify the 'scope' of the plugin. Scope is the name of the plugin normalized to a registration compatible form
+3. Identify the components that your plugin exposes
+   * These are usually the 'Component' or 'Routable' extensions that are registered in to the plugin
+   * These are the exports that are exposed from your plugin entrypoint file, usually `index.ts`
+4. Register the plugin on Custom Plugins page
```

Comment:
Where is the custom plugins page? I cannot find it in the UI.

Reply:
it's feature flagged
```
customPlugins:
  enabled: true
  federationEnabled: true
```